validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorTimingActions.java

```diff
@@ -97,8 +105,28 @@ public void onAttestationAggregationDue(final UInt64 slot) {
   }
 
   @Override
-  public void onAttesterSlashing(final AttesterSlashing attesterSlashing) {}
+  public void onAttesterSlashing(final AttesterSlashing attesterSlashing) {
+    delegates.forEach(delegates -> delegates.onAttesterSlashing(attesterSlashing));
+    final Set<UInt64> slashedIndices = attesterSlashing.getIntersectingValidatorIndices();
```

Comment:
I'd put all logic in `maybeValidatorSlashedAction.ifPresent(` here and in proposer slashing to not calculate this if it's not set. Who know why we could use this subscriptions in the future. And `validatorIndexProvider.getPublicKey` looks not cheap